packages/assets/src/scss/_variables.scss

```diff
@@ -15,4 +15,7 @@ $base-line-height: 1.5 !default;
 $border-radius-medium: calculateRem(8px) !default;
 $border-radius-large: calculateRem(12px) !default;
 
+$badge-size-small: calculateRem(12px) !default;
```

Comment:
you don't need to make these configurable as far as I remember my arrangements with Darek, even my `!default`s shouldn't be there :sweat_smile: 

In variables I would leave only variables that are shared between components